File: b_rank_skillcheck_archive/plural_form_02.cs
// 実行時間 20ms
using static System.Console;

internal class Program
{
    static void Main()
    {
        int n = int.Parse( ReadLine() );

        for( int _ = 0; _ < n; _++ ) {
            string a = ReadLine();

            if( a.EndsWith( "s" ) || a.EndsWith( "sh" ) || a.EndsWith( "ch" ) || a.EndsWith( "o" ) || a.EndsWith( "x" ) ) {
                WriteLine( $"{a}es" );
            } else {
                WriteLine( $"{a}s" );
            }
        }
        return;
    }
}

File: b_rank_skillcheck_archive/plural_form_02.py
# 実行時間 10ms
n = int(input())

for _ in [0] * n:
    a = input()

    if a.endswith("s") or a.endswith("sh") or a.endswith("ch") or a.endswith("o") or a.endswith("x"):
        print(f"{a}es")
    else:
        print(f"{a}s")
